test/xpu_api/tuple/tuple.tuple/tuple.elem/tuple_get_const_rvalue.pass.cpp

```diff
@@ -24,6 +24,19 @@
 
 class KernelGetConstRvTest;
 
+void
+host_test()
+{
+    using tuple_type = dpl::tuple<int>;
+
+    const tuple_type t(3);
```

Comment:
Probably, does sharing "a test body" (lines 32-37 here) between kernel code and host code does make sense?

Reply:
I made these changes only for testing why sometimes we have compile error here:
```C++
test/xpu_api/tuple/tuple.tuple/tuple.elem/tuple_get_const_rvalue.pass.cpp:39:31: error: static assertion failed due to requirement 'dpl::is_same<const int &&, const int &>::value'
   39 |                 static_assert(dpl::is_same<const int&&, decltype(dpl::get<0>(dpl::move(t)))>::value);
      |                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
test/xpu_api/tuple/tuple.tuple/tuple.elem/tuple_get_const_rvalue.pass.cpp:41:29: error: rvalue reference to type 'const int' cannot bind to lvalue of type 'const __tuple_element_t<0UL, tuple<int>>' (aka 'const int')
   41 |                 const int&& i = dpl::get<0>(dpl::move(t));
      |                             ^   ~~~~~~~~~~~~~~~~~~~~~~~~~
2 errors generated.
```

Reply:
This commented change already isn't actual - I added `TEST_GCC7_RVALUE_TUPLE_GET_BROKEN` macro to avoid compile error.